LICENSE.md

```diff
@@ -1,22 +1,234 @@
-There are ongoing discussions with the Colorado Department of State
-about which Open Source license is most appropriate for this system.
+The Colorado RLA Tool is Copyright (C) 2017 the Colorado Department of State, and is licensed under the [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html)  with a classpath exception.
 
-They have agreed to license it under a variant of the GPLv3, but the
-final decision about which variant it will be licensed under is still
-under discussion.
+GNU AFFERO GENERAL PUBLIC LICENSE
 
-In particular, should it be licensed under
-[AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) or
-[GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html)? Please see the
-FSF document, ["Why the Affero
-GPL"](https://www.gnu.org/licenses/why-affero-gpl.en.html), for a
-relevant discussion.
+Version 3, 19 November 2007
 
-If you, the reader, have input on this decision, please add a comment
-to [Issue #452](https://github.com/FreeAndFair/ColoradoRLA/issues/452).
+Copyright © 2007 Free Software Foundation, Inc. <https://fsf.org/> 
+Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.
 
-Thus, as of now, this system is not licensed for any use whatsoever
-beyond its owner, the Colorado Department of State.
+Preamble
 
+The GNU Affero General Public License is a free, copyleft license for software and other kinds of works, specifically designed to ensure cooperation with the community in the case of network server software.
+
+The licenses for most software and other practical works are designed to take away your freedom to share and change the works. By contrast, our General Public Licenses are intended to guarantee your freedom to share and change all versions of a program--to make sure it remains free software for all its users.
```

Comment:
Nit: it would be nice if the lines were wrapped to a reasonable length, like 80 characters.